plugins/catalog-backend/src/next/processing/util.ts

```diff
@@ -76,6 +77,10 @@ export function toAbsoluteUrl(
   }
 }
 
+export function isObject(value: JsonValue | undefined): value is JsonObject {
+  return typeof value === 'object' && value !== null && !Array.isArray(value);
```

Comment:
`lodash.isPlainObject` is even stricter, would it make sense to just call out to that instead? https://github.com/lodash/lodash/blob/master/isPlainObject.js

For example, the above code returns true for `new Map()`, was that the intention?

Reply:
Yep it's fine because the input is already `JsonValue`, so `Map` isn't allowed. Lodash doesn't narrow the type enough as `isObject` guarantees that it's an `object`, but that includes arrays. It would be nice to use `isPlainObject` but it doesn't have a type guard at all so kinda defeats the purpose.

Reply:
Yeah I'd meant to keep this function for its narrowing, but to implement it in terms of `isPlainObject`. Well ok if we can't get a Map accidentally then I guess it doesn't matter much.